src/design-system-team.md.njk

```diff
@@ -23,6 +23,7 @@ If you want to contact the team you can
 - Imran Hussain – Community Manager
 - Izabela Podralska – Delivery Manager
 - Joe Lanman – Lead Interaction Designer
+- Katrina Birch - User Researcher
```

Comment:
```suggestion
- Katrina Birch – User Researcher
```

Reply:
Suggesting this change as we insert dashes between the name and the person's role.

Reply:
@SaraC19 Eoin's comment here is about making sure we use the right kind of dash to be consistent. You should be able to press 'Commit suggestion' and 'Commit changes' to accept this suggestion - are you ok to do that? Happy to walk you through it.

Reply:
Ah ha - thank you! I'll press some buttons now and will see if it works.